assembl/graphql/vote_session.py

```diff
@@ -168,6 +169,12 @@ def resolve_vote_spec_template_id(self, args, context, info):
     def resolve_vote_type(self, args, context, info):
         return self.type
 
+    def resolve_my_votes(self, args, context, info):
+        user_id = context.authenticated_userid
+        # use votes_of(user_id) instead of votes_of_current_user because
+        # request threadlocal is not properly set in tests
```

Comment:
We have a fixture for this exactly like: `test_adminuser_webrequest`

Reply:
graphql_request already uses test_adminuser_webrequest, but this doesn't push registry, request to the pyramid threadlocal manager for use with get_current_request()